src/projects/TodoList/components/CreateNewCategory/index.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import './style.css';
+
+class CreateNewCategory extends React.Component {
+  render () {
+    // logic goes here
+
+    let rgb = [];
+    rgb.push(this.props.newCategoryRed, this.props.newCategoryGreen, this.props.newCategoryBlue)
```

Comment:
Rather than creating an empty array inside a variable then pushing the 3 items into it, you can just initialise the array with the items inside:

```js
let rgb = [this.props.newCategoryRed, this.props.newCategoryGreen, this.props.newCategoryBlue]
```

Reply:
ahhh damn of course. will change

Reply:
done